qml/components/TimeProgressBar.qml

```diff
@@ -21,17 +21,42 @@
 ********************************************/
 
 import QtQuick 2.12
+import QtQuick.Layouts 1.3
 import "../style"
 import "helpers"
 
 Item {
+    id: root
+
     property int minValue: 0
     property int maxValue: 100
     property int value: 0
 
+    readonly property bool timeEditionInProgress: textEditableInput.visible
```

Comment:
I would prefer something like `editMode` here and `setEditMode(edit)` instead of `startTimeEdition` and `endTimeEdition`.
What do you think?

Reply:
I've changed it and used property `timeEditMode` instead. Could you take a look?